VKProject/Src/Models/User.cs

```diff
@@ -0,0 +1,14 @@
+namespace VKProject.Models;
+
+public class User
+{
+    public string? login { get; set; }
```

Comment:
why lowercase? you can use json attribute to say the compillir how the property called in response

Reply:
done